streams/id.m3u

```diff
@@ -99,6 +107,8 @@ https://h1.intechmedia.net/intech/ch29.m3u8
 #EXTINF:-1 tvg-id="JayaTVJayapura.id" user-agent="Mozilla/5.0 (X11; Linux x86_64)",Jaya TV (360p) [Geo-blocked]
 #EXTVLCOPT:http-user-agent=Mozilla/5.0 (X11; Linux x86_64)
 https://h1.intechmedia.net/intech/ch52.m3u8
+#EXTINF:-1 tvg-id="JekTV.id",Jek TV (720p)
+http://61.8.64.138:1935/jektv/myStream/playlist.m3u8
```

Comment:
Offline.

<img width="381" alt="image" src="https://github.com/iptv-org/iptv/assets/7253922/94ca5c74-d216-4e13-a880-38f280785ee1">

Reply:
Also seems to be from an official source : https://jektv.co.id/live/